main.py

```diff
@@ -152,14 +152,14 @@
     redAllianceTeamNums = tbaWrapper.getAllianceTeamNums(matchNum, 'red')
     blueAllianceTeamNums = tbaWrapper.getAllianceTeamNums(matchNum, 'blue')
 
-    for team in redAllianceTeamNums:
-        red_correction = next((item for item in correctionsDataRaw if int(item['teamNum']) == team and int(item['matchNum']) == matchNum), None)
-        if red_correction:
-            break
-    for team in blueAllianceTeamNums:
-        blue_correction = next((item for item in correctionsDataRaw if int(item['teamNum']) == team and int(item['matchNum']) == matchNum), None)
-        if blue_correction:
-            break
+    red_corrections = {}
+    blue_corrections = {}
+
+    for correction in correctionsDataRaw:
+        if int(correction['matchNum'] == matchNum) and int(correction['teamNum']) in redAllianceTeamNums:
```

Comment:
It should be int(correction['matchNum']), the first close parenthesis is in the wrong spot.

Reply:
Same for the blue corrections